test/e2e-helm/e2e_helm_suite_test.go

```diff
@@ -45,8 +47,16 @@ var _ = BeforeSuite(func() {
 	tc, err = testutils.NewTestContext(testutils.BinaryName, "GO111MODULE=on")
 	Expect(err).NotTo(HaveOccurred())
 
-	By("creating a new directory")
-	Expect(tc.Prepare()).To(Succeed())
+	tc.Domain = "example.com"
+	tc.Group = "cache"
+	tc.Version = "v1alpha1"
+	tc.Kind = "Memcached"
+	tc.Resources = "memcacheds"
+	tc.ProjectName = "memcached-operator"
+	tc.Kubectl.Namespace = fmt.Sprintf("%s-system", tc.ProjectName)
+
```

Comment:
```suggestion
By("creating the repository")
Expect(tc.Prepare()).To(Succeed())
```

To create the e2e-1234 dir.